.github/workflows/conventional-commits-check.yaml

```diff
@@ -0,0 +1,49 @@
+name: Conventional Commits Check
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+
+jobs:
+  check-conventional-commits:
+    name: Conventional Commits
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - name: Check Commit Conventions
+        uses: webiny/action-conventional-commits@v1.3.0
```

Comment:
I do not particularly care about individual commits, as the whole PR will be squashed anyway. Please remove this step.